docker-compose.prod.yml

```diff
@@ -25,6 +25,8 @@ services:
       - POSTGRES_USER=${POSTGRES_USER:?required}
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:?required}
       - FORTYTWO_APP_CALLBACK_URL=${FORTYTWO_APP_CALLBACK_URL:?required}
+      - WEBSITE_OWNER_PASSWORD=${WEBSITE_OWNER_PASSWORD:?required}
+      - WEBSITE_OWNER_USERNAME=${WEBSITE_OWNER_USERNAME:?required}
```

Comment:
suggestion: We could add the `WEBSITE_OWNER_*` to the deployment section of the README

Reply:
done